### PR TITLE
Correct docblock CartTotalRepository get method

### PR DESCRIPTION
**Pull request by mrtuvn**
### Description (*)
Just correct return class of get method

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by mrtuvn** [comment]
@lbajsarowicz i have updated new file ui test for changes

**Comment by mrtuvn** [comment]
@lbajsarowicz this one can you take a look

File: app/code/Magento/Quote/Test/Unit/Model/Cart/CartTotalRepositoryTest.php [hunk 1]
```diff
@@ -80,20 +117,20 @@ protected function setUp()
                 'getItemsQty',
                 'collectTotals'
             ]);
-        $this->quoteRepositoryMock = $this->createMock(\Magento\Quote\Api\CartRepositoryInterface::class);
+        $this->quoteRepositoryMock = $this->createMock(CartRepositoryInterface::class);
         $this->addressMock = $this->createPartialMock(
-            \Magento\Quote\Model\Quote\Address::class,
+            Address::class,
             ['getData', 'getTotals']
         );
```
Comment (lbajsarowicz) [inline app/code/Magento/Quote/Test/Unit/Model/Cart/CartTotalRepositoryTest.php]: You can keep this one line (< 80 chars)

File: app/code/Magento/Quote/Model/Cart/CartTotalRepository.php [hunk 2]
```diff
@@ -78,12 +83,13 @@ public function __construct(
...
     /**
-     * @inheritdoc
+     * Get cart total repository
      *
-     * @param int $cartId The cart ID.
-     * @return Totals Quote totals data.
+     * @param int $cartId
+     * @return QuoteTotalsInterface
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
      */
```
Comment (lbajsarowicz) [inline app/code/Magento/Quote/Model/Cart/CartTotalRepository.php]: Why not just `@inheritdoc`? The interface seems to have it well-declared.

File: app/code/Magento/Quote/Model/Cart/CartTotalRepository.php [hunk 3]
```diff
@@ -96,12 +102,12 @@ public function get($cartId)
...
+        /** @var QuoteTotalsInterface $quoteTotals */
         $quoteTotals = $this->totalsFactory->create();
         $this->dataObjectHelper->populateWithArray(
             $quoteTotals,
             $addressTotalsData,
-            \Magento\Quote\Api\Data\TotalsInterface::class
+            QuoteTotalsInterface::class
         );
         $items = [];
         foreach ($quote->getAllVisibleItems() as $index => $item) {
```
Comment (lbajsarowicz) [inline app/code/Magento/Quote/Model/Cart/CartTotalRepository.php]: Perfect case to replace with 
```php
$items = array_map([$this->itemConverter, 'modelToDataObject'], $this->getAllVisibleItems())
```

File: app/code/Magento/Quote/Test/Unit/Model/Cart/CartTotalRepositoryTest.php [hunk 4]
```diff
@@ -4,73 +4,110 @@
...
+
+    /**
+     * @var string
+     */
+    private const STUB_CURRENCY_CODE = 'en_US';
+
+    /**
+     * @var string
+     */
+    private const STUB_COUPON = 'coupon';
```
Comment (lbajsarowicz) [inline app/code/Magento/Quote/Test/Unit/Model/Cart/CartTotalRepositoryTest.php]: `const` != `@var`
Remove all the doc blocks for these consts.

File: app/code/Magento/Quote/Test/Unit/Model/Cart/CartTotalRepositoryTest.php [hunk 5]
```diff
@@ -4,73 +4,110 @@
...
+     * @var TotalsConverter|MockObject
      */
     protected $totalsConverterMock;
 
     protected function setUp()
     {
-        $this->objectManager = new ObjectManager($this);
+        $this->objectManager = new ObjectManagerHelper($this);
         $this->totalsFactoryMock = $this->createPartialMock(
             \Magento\Quote\Api\Data\TotalsInterfaceFactory::class,
```
Comment (lbajsarowicz) [inline app/code/Magento/Quote/Test/Unit/Model/Cart/CartTotalRepositoryTest.php]: Import class, move to same line.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Still, some work has to be done.

**Review by lbajsarowicz — APPROVED** [review]
:heavy_check_mark: Thank you for your contribution.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7183](https://jira.corp.magento.com/browse/ENGCOM-7183) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Comment by engcom-Kilo** [comment]
Failed functional tests not related to the changes in this PR

**Pull request merged**